### ISSUE TITLE
Handle out of date clients

### ISSUE THREAD

**Issue by julianpoy** [Kondeo/ccra-book#40]
We need to leave room for a potential situation where the client has a very old software version, while the server is newer. We cannot always operate backwards-compatible, so this means we need to find a way to disable clients that are too out-of-date.

Heres how we do it:
Every request to the server should include the local app version number. The local app will have an app-wide error handler for 426 and will display any message sent to the server for that code. The same thing goes for 101, which is the suggested upgrade notice.

TL;DR:
426 - Failure. Display server message. Required upgrade.
101 - Success. Display server message. Client will soon be too-far out of date.

**Comment by julianpoy** [comment]
Please integrate this into the current handler for version numbers. Upon response, if you get either of these codes display the response message. Please ask questions if you do not understand, because I've not explained this issue very well.

**Comment by julianpoy** [comment]
Closed via #99
